benchmarks/bytes_and_flops/bench_unroll_stride.hpp

```diff
@@ -87,25 +87,25 @@ struct Run<Scalar, UNROLL, STRIDE> {
                     C(n, i, 0) = a1;
 #endif
 #if (UNROLL == 2)
-                    C(n, i, 0) = a1 + a2;
+                    C(n, iter, 0) = a1 + a2;
```

Comment:
What about L87?
Also that does not look right to me.  It gives out-of-bounds accesses if the number of iterations exceeds the view extent in this dimension.  Also it changes the meaning of the code.

Reply:
yeah this all needs to stay `i` that said the rename of the shadowed variable is probably a good idea :-)

Reply:
Oups, very sorry but thank you for this good catch @dalg24.